lib/api.js

```diff
@@ -42,6 +42,62 @@ module.exports = {
         })
     },
 
+    getLastTransaction: function (address, endpoint) {
```

Comment:
You function is called `getLastTransaction` but you only fetch the address, so it may be better to use `getLastTransactionAddress`

Reply:
I updated this function in this commit
https://github.com/archethic-foundation/libjs/pull/10/commits/1dae13f46b32a8e221658630dac9d6c4ac2fc7e8
but it doesn't pass CodeFactor verification on a file I didn't update.
Do I have to correct CodeFactor issues on crypto.js file ?